packages/paste-core/components/breadcrumb/__tests__/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {axe} from 'jest-axe';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+expect.extend(matchers);
+
+describe('Breadcrumb', () => {
+  it('it should render a nav with correct aria-label', () => {
+    const {getByRole} = render(
+      <Breadcrumb>
+        <BreadcrumbItem href="#">foo</BreadcrumbItem>
+        <BreadcrumbItem>bar</BreadcrumbItem>
+      </Breadcrumb>
+    );
+    expect(getByRole('navigation').getAttribute('aria-label')).toBe('breadcrumb');
+  });
+
+  it('it should render a list with display: inline-flex', () => {
+    render(
+      <Breadcrumb>
+        <BreadcrumbItem href="#">foo</BreadcrumbItem>
+        <BreadcrumbItem>bar</BreadcrumbItem>
+      </Breadcrumb>
+    );
+    const renderedBreadcrumb = screen.getByRole('list');
```

Comment:
nit: all of these constants change their meaning. For example, this isn't really `renderedBreadcrumb`, it's the rendered list. It might be nice to have explicit names so the test code reads better.

Reply:
Yep. Totally makes sense.